command/certificate/format.go

```diff
@@ -51,12 +64,72 @@ Convert PEM format to DER and write to disk:
 '''
 $ step certificate format foo.pem --out foo.der
 '''
+
+Convert a .p12 file to a certificate and private key:
+
+'''
+$ step certificate format foo.p12 --crt foo.crt --key foo.key --format pem
+'''
+
+Convert a .p12 file to a certificate, private key and intermediate certificates:
+
+'''
+$ step certificate format foo.p12 --crt foo.crt --key foo.key --ca intermediate.crt --format pem
+'''
```

Comment:
It looks like if the flag `--format` is not passed it will write PEM files, as it should be. We should show this in one of these two examples, explaining that is the default behavior.